### PR TITLE
Updates and corrections to rate limit pages

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Make corrections to rate limits pages
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-416973](https://oktainc.atlassian.net/browse/OKTA-416973)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 1]
```diff
@@ -37,16 +35,14 @@ If an org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
...
 
-* [Client-based rate limits](/docs/reference/rl-clientbased/): Client-based rate limiting for the `/authorize` endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the `/authorize` endpoint. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+* [Client-based rate limits](/docs/reference/rl-clientbased/): Client-based rate limiting uses a combination of the client ID, user's IP address, and Okta device identifier (for the `/oauth2/v1/authorize` endpoint) or the IP address and device identifier (for the `/login/login.htm` endpoint) to provide granular isolation for requests made to the `/oauth2/v1/authorize` or `/login/login.htm` endpoint. This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:38]: I find this sentence a bit hard to follow. Perhaps another bullet list, one level lower, could help?

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 2]
```diff
@@ -21,13 +21,14 @@ These limits are part of the Okta [Rate limit](/docs/reference/rate-limits) poli
 > **Note:**
 >
 > * In addition to the rate limits listed on this page, Okta applies rate limits per API, divided into three categories. See the [Rate limit overview](/docs/reference/rate-limits).
+> * [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/) apply to a variety of endpoints across different APIs for customers that purchased this add-on.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:24]: Should this be present tense?

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 3]
```diff
@@ -21,13 +21,14 @@ These limits are part of the Okta [Rate limit](/docs/reference/rate-limits) poli
...
 > * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
 
 ## Concurrent rate limits
 
-To protect the service for all customers, Okta enforces concurrent rate limits, which is a limit on the number of simultaneous transactions. Concurrent rate limits are distinct from [the org-wide, per-minute API rate limits](/docs/reference/rate-limits/), which measure the total number of transactions per minute. Transactions are typically very short-lived. Even very large bulk loads rarely use more than 10 simultaneous transactions at a time.
+To protect the service for all customers, Okta enforces concurrent rate limits, which is a limit on the number of simultaneous transactions. Concurrent rate limits are distinct from [the org-wide, per-minute API rate limits](/docs/reference/rate-limits/), which measure the total number of transactions per minute. Transactions are typically very short-lived. Even very large bulk loads rarely require more than 10 simultaneous transactions.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:31]: Is the use of the adverb "very" necessary here?

File: packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md [hunk 4]
```diff
@@ -23,37 +23,35 @@ For example:
...
+The three headers behave a little differently for concurrent rate limits: when the number of unfinished requests is below the concurrent rate limit, request headers only report org-wide rate limits. When you exceed a concurrent rate limit threshold, the headers report that the limit has been exceeded. When you drop back down below the concurrent rate limit, the headers switch back to reporting the time-based rate limits. Additionally, the `X-Rate-Limit-Reset` time for concurrent rate limits is only an estimate. There's no guarantee that enough requests will complete to stop exceeding the concurrent rate limit at the time indicated.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md:35]: Perhaps this paragraph would benefit from being in a bullet list?

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 5]
```diff
@@ -8,25 +8,27 @@ excerpt: >-
...
 
-For example, example.com has 10 OAuth 2.0 applications running in a production environment. Bob's team is launching a new marketing portal that is a single page OAuth 2.0 application. Unaware of the rate limits on the `/authorize` endpoint, Bob's team begins running some batch testing scripts against the newly created application that makes hundreds of `/authorize` requests in a single minute. Without the client-based rate limit framework, the new marketing portal application could potentially consume all of the per minute request limits assigned to example.okta.com and thereby cause rate limit violations for the rest of the users that access the other OAuth 2.0 applications.
+Curently, you cannot update the per client rate limit. Every client ID/IP/device identifier combination (for the /authorize endpoint) or IP/device identifier combination (for the /login/login.htm endpoint) gets 60 total requests per minute and a maximum of two concurrent requests.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:24]: Should the endpoints in this paragraph be in monospace?

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 6]
```diff
@@ -8,25 +8,27 @@ excerpt: >-
...
+
+For example, example.com has 10 OAuth 2.0 applications running in a production environment. Bob's team is launching a new marketing portal that is a single page OAuth 2.0 application. Unaware of the rate limits on the `/oauth2/v1/authorize` endpoint, Bob's team begins running some batch testing scripts against the newly created application that makes hundreds of `/oauth2/v1/authorize` requests in a single minute. Without the client-based rate limit framework, the new marketing portal application could potentially consume all of the per minute request limits assigned to example.okta.com and thereby cause rate limit violations for the rest of the users that access the other OAuth 2.0 applications.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:26]: When an adjective, should it be "single-page"?

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 7]
```diff
@@ -12,12 +12,12 @@ This page provides the API rate limits for authentication and end-user activitie
 > **Note:**
 >
 > * To learn more about rate limits, visit our [overview](/docs/reference/rate-limits) and [best practices](/docs/reference/rl-best-practices) pages.
+
 > * In addition to the rate limit per API, Okta implements limits on concurrent requests, Okta-generated email messages, end user requests, and home page endpoints. These limits are described on the [Additional limits](/docs/reference/rl-additional-limits/) page.
+> * [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/) apply to a variety of endpoints across different APIs for customers that purchased this add-on.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:17]: Present tense?

**Review by brentschaus-okta — APPROVED** [review]
LGTM (just a few nits and questions)

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md (hunk 2 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:24]: This should be OK as is. Our customers must have previously purchased the add-on before they can apply dynamic scale rate limits.

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:38]: I reworded and shortened the sentence the best I could. Was going to create a sub-bullet list, but noticed that it would stand out in that particular section of the page.

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md (hunk 3 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:31]: Fixed

File: packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md (hunk 4 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md:35]: Fixed.

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md (hunk 5 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:24]: good catch! fixed.

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md (hunk 6 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:26]: 👍

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 8]
```diff
@@ -138,29 +138,29 @@ To configure the client-based rate limit for existing orgs:
...
 
-No. Today every client ID/IP/device identifier combination (for the `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of two concurrent requests.
+No. Today every client ID/IP/device identifier combination (for the OAuth 2.0 `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of two concurrent requests.
 
 **Q: Does the org-wide rate limit still apply when I enable the client-based rate limit?**
 
 Yes. When the cumulative total request or maximum concurrent requests from every unique client exceeds the org-wide rate limits, your Okta org experiences org-wide rate limit errors.
 
-**Q: Would the rate limit headers returned by Okta on the /authorize or /login/login.htm endpoint reflect client-specific rate limits?**
+**Q: Would the rate limit headers returned by Okta on the /oauth2/v1/authorize or /login/login.htm endpoint reflect client-specific rate limits?**
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:161]: In this line and in a few others as well we have a specific version of the authorize endpoint listed, rather than the generic "OAuth 2.0 /authorize" mention that we're using elsewhere. Shouldn't it be the latter?

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 9]
```diff
@@ -8,25 +8,27 @@ excerpt: >-
...
 
 Currently, the client-based rate limits apply to the `/login/login.htm` endpoint and OAuth 2.0 `/authorize` endpoint on both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share a [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
 
-Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket, for example, (`/oauth2/${authorizationServerId}/v1` (Custom Authorization Server) or `/oauth2/v1` (Okta Org Authorization Server). The per minute rate limits on these endpoints apply across an Okta tenant.
+Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket, for example, OAuth 2.0 `/authorize` (Custom Authorization Server) or `/oauth2/v1` (Okta Org Authorization Server). The per minute rate limits on these endpoints apply across an Okta tenant.
```
[the remark this answers is not in the document]
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:22]: Fixed

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md (hunk 8 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md]: Fixed.

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 10]
```diff
@@ -37,16 +37,14 @@ If an org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
...
 
-* [Client-based rate limits](/docs/reference/rl-clientbased/): Client-based rate limiting for the `/authorize` endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the `/authorize` endpoint. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+* [Client-based rate limits](/docs/reference/rl-clientbased/): To provide granular isolation, client-based rate limiting uses a combination of the client ID/IP address/device identifier for requests made to the OAuth 2.0 `/authorize` endpoint or the IP address/device identifier for requests made to the `/login/login.htm` endpoint. This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:40]: "This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations."
->
"This framework isolates OAuth 2.0 clients that are generating unexpected traffic, thereby ensuring valid users and applications don't run into rate limit violations."

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 11]
```diff
@@ -14,15 +14,15 @@ The Okta API rate limits are divided into three categories: authentication/end u
 
 To access the individual API limits, visit a category page by clicking the appropriate category link in the table.
 
-> We enforce limits at the individual API endpoint level **as requests per minute**.
+> We enforce **per-minute** limits at the individual API endpoint level.
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:17]: The part that I think we need to change is "at the individual API endpoint level"
Some limits are for one individual API and some are for multiple. 
Maybe we should just say "We enforce the following per-minute limits:"

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 12]
```diff
@@ -36,14 +38,14 @@ The best way to describe how client-based rate limiting works is to provide some
 
 #### Client-based isolation for a unique IP address
 
-##### Example of using the /authorize request
+##### Example of using the /oauth2/v1/authorize request
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:41]: one more to fix here

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 13]
```diff
@@ -57,7 +59,7 @@ When Bob and Alice try to navigate to their application and land on the sign-in
 
 If the request quota isn't set, Bob or Alice would have the ability to make the request enough times that the entire org bucket would hit rate limits that would affect the entire org.
 
-#### Client-based isolation for users accessing /authorize from a NAT IP
+#### Client-based isolation for users accessing /oauth2/v1/authorize from a NAT IP
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:62]: make generic

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md (hunk 11 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:17]: Fixed this one and the three other instances of this.

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md (hunk 10 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:40]: Fixed.

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md (hunk 12 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:41]: Fixed

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md (hunk 13 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:62]: Fixed.

**Review by peteryaworsky-okta — APPROVED** [review]
